README.md

```diff
@@ -60,217 +81,67 @@ increase security by using a custom `documentLoader` that is similarly strict
 and will only load a subset of documents that is constrained by some technical,
 security, or business rules.
 
-Install with npm:
```

Comment:
Why are the common case npm install instructions gone in favor of only development checkout instructions?  I highly suggest we add back the common case here so people can easily see what npm package to use.

Reply:
Reading more... this is because it's suggested to use the other suites vs this directly?  I think the text ordering may be a bit confusing in that regard.

Reply:
Yeah, because most devs won't be installing jsigs directly. (hopefully).